src/Parallel/Main.hpp

```diff
@@ -55,9 +57,6 @@ class Main : public CBase_Main<Metavariables> {
   using phase_change_tags_and_combines =
```

Comment:
how about renaming this to `phase_change_tags_and_combines_list` or `list_of_phase...` so that it doesn't sound as verby?